jobs/build/publish-rpms/Jenkinsfile

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env groovy
+
+node {
+    checkout scm
+    buildlib = load( "pipeline-scripts/buildlib.groovy" )
+    commonlib = buildlib.commonlib
+    commonlib.describeJob("publish-rpms", """
+        <h2>Automatically publishing rpms to mirror</h2>
+    """)
+
+    properties(
+        [
+            disableResume(),
+            buildDiscarder(
+                logRotator(
+                    artifactDaysToKeepStr: '',
+                    artifactNumToKeepStr: '',
+                    daysToKeepStr: '',
+                    numToKeepStr: ''
+                )
+            ),
+            [
+                $class : 'ParametersDefinitionProperty',
+                parameterDefinitions: [
+                    string(
+                        name: 'BUILD_VERSION',
+                        description: 'Target release version',
+                        trim: true,
+                    ),
+                ]
+            ],
+            disableResume(),
+            disableConcurrentBuilds()
+        ]
+    )
+    commonlib.checkMock()
```

Comment:
you need `commonlib.mockParam()` above in the params too or this will always fail.